src/components/FormField/Input/PasswordInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { Icon } from "../..";
+import { PasswordInputProps } from "./types";
+import { IconButton } from "./InputStyledComponents";
+import Container from "./Container";
+
+export const PasswordInput = ({ type, disabled, ...props }: PasswordInputProps) => {
+  const [viewPassword, setViewPassword] = useState<boolean>(false);
+  const togglePasswordViewer = () => {
+    setViewPassword((viewPassword: boolean) => !viewPassword);
+  };
+
+  return (
+    <Container
+      type="password"
+      disabled={disabled}
+      {...props}
+    >
+      <IconButton
```

Comment:
I don't see the real input here

Reply:
The real input has been moved inside the Container so that the code remains in one place
Additional values are added as children